include/nil/crypto3/zk/snark/systems/plonk/placeholder/permutation_argument.hpp

```diff
@@ -69,17 +69,14 @@ namespace nil {
                     };
 
                     static inline math::polynomial_dfs<typename FieldType::value_type> polynomial_product(
-                        std::vector<math::polynomial_dfs<typename FieldType::value_type>> multipliers)
+                        std::vector<math::polynomial_dfs<typename FieldType::value_type>>& multipliers)

```

Comment:
We don't want to change the values of multipliers, if it's used outside this function. (This function totally changes the values, nothing is left from the initial values).  If it's not used outside this function, then we can move the value inside this function. I don't think we must use a reference here.

Reply:
updated in 64f7f7a3